Source/CesiumRuntime/Private/CesiumGltfComponent.cpp

```diff
@@ -1172,6 +1172,35 @@ std::string getPrimitiveName(
   }
   return name;
 }
+
+/// Helper used to log only once per unsupported primitive mode.
+struct PrimModeLogHelper {
+  std::array<std::atomic_bool, (size_t)MeshPrimitive::Mode::TRIANGLE_FAN + 1>
+      alreadyLogged_;
```

Comment:
Nitpick: Remove the underscore? Unless it's there for a reason.

```suggestion
      alreadyLogged;
```